story3.js

```diff
@@ -0,0 +1,73 @@
+fetch('./films.json')
+    .then((response) => response.json())
+    .then((json) => {
+        let  films = json.films
+
+        function shuffleArray(array) {
+            for (var i = array.length - 1; i > 0; i--) {
+                var j = Math.floor(Math.random() * (i + 1));
+                var temp = array[i];
+                array[i] = array[j];
+                array[j] = temp;
+            }
+        }
```

Comment:
It would be nicer to move this function to outside the whole `fetch` block, to make it more reusable